pathfinder/api/rest.py

```diff
@@ -46,10 +46,9 @@ def _validate_token_network_argument(
 
     def _validate_channel_id_argument(self, channel_id: str) -> Optional[Tuple[Dict, int]]:
```

Comment:
just use ints, no need for silly conversions at this level - they should be done by the caller.

Reply:
(you can use something like

`self.api.add_resource(Resource, API_PATH + "/channels/<str:token_addr>/<int:channel_id>")`

when setting up endpoints)

Reply:
fair point, I haven't considered this scenario.